gql_schema_codegen/block/block.py

```diff
@@ -18,6 +20,14 @@ class BlockFieldInfo(NamedTuple):
     value_type: str
 
 
+# a dictionary where for each node, we hold its children
+inheritanceTree: Dict[str, Set[str]] = {"root": set()}
```

Comment:
```suggestion
inheritanceTree: Dict[str, Set[str]] = defaultdict(lambda: set())
```

https://docs.python.org/3.11/library/collections.html#collections.defaultdict

then whatever you access on the dict is always present automatically